portability-transfer/src/main/java/org/datatransferproject/transfer/JobProcessor.java

```diff
@@ -159,6 +177,8 @@ void processJob() {
           EventCode.WORKER_JOB_FINISHED);
       addErrorsAndMarkJobFinished(jobId, success, loggedErrors);
       hooks.jobFinished(jobId, success);
+      SignalType finalStatus = success ? SignalType.JOB_COMPLETED : SignalType.JOB_ERRORED;
```

Comment:
note: looking at `success` here is probably getting into the weeds of a more nuanced state then we need to for your signals-change. perhaps drop this to be a static `JOB_DONE` (and/or incorporate the other comment I've made above directly on the signals enum)

----

(this is more a discussion for the SignalType enum, but making it here for clarity)

replace `JOB_COMPLETED` `JOB_PERFECT_TRANSFER` and/or replace `JOB_ERRORED` with `JOB_COMPLETED_PARTIAL`

also: we can tell if something was _skipped_ if `success && !loggedErrors.isEmpty()`


tl;dr replace `COMPLETED` with `DONE` and have a suffix to capture nuance (eg: `_PARTIAL` or `_PERFECT`).